content/logs/attributes_naming_convention.md

```diff
@@ -0,0 +1,139 @@
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `network.client_ip` | `string` | The IP address of the client which initiated the TCP connection. |
+| `network.destination_ip` | `string` | The IP address the client connected to. |
+| `network.client_port` | `string` | The port of the client which initiated the connection. |
+| `network.destination_port` | `string` | The TCP port the client connected to. |
+| `network.bytes_read` | `string` | Total number of bytes transmitted from the client to the server when the log is emitted. |
+| `network.bytes_write` | `string` | Total number of bytes transmitted from the server to the client when the log is emitted. |
+
+Typical integrations relying on these: Apache, Varnish, AWS ELB, Nginx, HAPROXY, etc.
+
+### HTTP Requests
+
+Related to the data commonly used in HTTP requests & accesses. All attributes are prefixed by `http`.
+
+Typical integrations relying on these attributes are: Apache, Rails, AWS CloudFront, web applications servers, etc...
+
+#### Common attributes
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `http.url` | `string` | The URL of the HTTP request. |
+| `http.status_code` | `number` | The IP address the client connected to. |
+| `http.method` | `string` | The HTTP verb of the request.|
+| `http.referer` | `string` | The HTTP referer. |
+| `http.request_id` | `string` | The HTTP request id. |
+| `http.user_agent` | `string` | The User-Agent as it is sent (raw format). See bellow for all details about it. |
+
+#### URL details attributes
+
+Details about the parsed parts of the HTTP url. Generally generated thanks to the [URL parser](https://docs.datadoghq.com/logs/processing/#url-parser).
+
+|                                                 |                                     |                          |           
+| :---                                            | :---                                 | :----                    |         
+| **Fullname**                                     | **Type**                           | **Description**|
+| `http.url_details.host` | `string` | The HTTP host part of the url. |
+| `http.url_details.port` | `number` | The HTTP port part of the url. |
+| `http.url_details.path` | `string` | The HTTP path part of the url. |
+| `http.url_details.method` | `string` | The HTTP method part of the url. |
```

Comment:
method does not exist in the url_details.

However `scheme` does:

`http.url_details.scheme` => The protocol name of the URL (http or https)